pkg/asset/manifests/vsphere/cloudproviderconfig.go

```diff
@@ -16,6 +16,8 @@ func printIfNotEmpty(buf *bytes.Buffer, k, v string) {
 // CloudProviderConfig generates the cloud provider config for the vSphere platform.
 // folderPath is the absolute path to the VM folder that will be used for installation.
 // p is the vSphere platform struct.
+// We check to see if a port value is specified for vcenter("url:port") and send them as
+// separate fields in cloud config.
```

Comment:
nit: I'm not sure if this comment belongs here for the function-level documentation. also it probably would belong before the list/description of parameters.